contracts/TestDaggerHashPrecompile.sol

```diff
@@ -49,6 +50,9 @@ contract TestKVWithDaggerHash is TestDecentralizedKV {
         assembly {
             paddrValue := sload(paddr.slot)
         }
+        // This optimization is only working for we have a PhyAddr with byte32 or less length
+        // To use Merkle tree commitment, we extend a hash field from Bytes24 to Bytes32
+        // This one immediately goes invalid 
```

Comment:
@qizhou Is this comment stating right? Now a PhyAddr is larger than Bytes32, the gas-saving optimization seems not valid anymore?